shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeIngressServiceImpl.java

```diff
@@ -160,6 +160,7 @@ public void createOrUpdateBridgeIngressSecret(BridgeIngress bridgeIngress, Bridg
         expected.getData().put(GlobalConfigurationsConstants.KAFKA_CLIENT_SECRET_ENV_VAR, Base64.getEncoder().encodeToString(bridgeDTO.getKafkaConnection().getClientSecret().getBytes()));
         expected.getData().put(GlobalConfigurationsConstants.KAFKA_SECURITY_PROTOCOL_ENV_VAR, Base64.getEncoder().encodeToString(bridgeDTO.getKafkaConnection().getSecurityProtocol().getBytes()));
         expected.getData().put(GlobalConfigurationsConstants.KAFKA_TOPIC_ENV_VAR, Base64.getEncoder().encodeToString(bridgeDTO.getKafkaConnection().getTopic().getBytes()));
+        expected.getData().put(GlobalConfigurationsConstants.KAFKA_ERRORS_TOPIC_ENV_VAR, Base64.getEncoder().encodeToString(bridgeDTO.getKafkaConnection().getErrorsTopic().getBytes()));
```

Comment:
I pass the `error-handler` topic name to `BridgeIngress` so that they can read the error messages.. this will however change when KNative is merged as the implementation will move to a Sidecar.. the intent however remains the same.

Reply:
This can be probably removed now that we are using a different processor for the error handling

Reply:
Yes, you are correct. Could you please create a JIRA so it's not forgotten and add to the EPIC MGDOBR-144, I think.